core/config.py

```diff
@@ -22,5 +22,5 @@
     from types_.config import Config
 
 
-with open("config.toml", "rb") as fp:
+with open(".config.toml", "rb") as fp:
```

Comment:
Why are we changing the file path?

Reply:
i think it was to hide the .config.toml file from vs-code project view

Reply:
Ah that makes sense lol